src/julia_gc.c

```diff
@@ -474,6 +474,20 @@ static void TryMark(void * p)
     }
 }
 
+static void TryMarkRangeReverse(void * start, void * end)
+{
+    if (lt_ptr(end, start)) {
+        SWAP(void *, start, end);
+    }
+    char * p = (char *)(align_ptr(start));
```

Comment:
```suggestion
    char * p = align_ptr(start);
```
to match `TryMarkRange`

Reply:
@rbehrends You can accept these proposed changes with a single click into your PR -- try it?

Reply:
Alternatively, either apply the change manually, or comment here that it is undesirable, and mark this conversation as resolved?

Reply:
I actually kept this an explicit cast and made the other one an explicit cast, too.